cmd/otelcontribcol/components.go

```diff
@@ -41,6 +41,7 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsxrayreceiver"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/collectdreceiver"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/dockerstatsreceiver"
```

Comment:
Please split the adding the component to the binary, vs other changes to the component in separate PRs.

Reply:
It appears to be common practice for new components to be included here out of the gate.  If this is a policy we need to enforce I think it should be clearly documented or even warrant automation.

Reply:
removed